nixos/modules/services/web-apps/lasuite-docs.nix

```diff
@@ -0,0 +1,473 @@
+      });
+
+    systemd.services.lasuite-docs = {
+      description = "Docs from SuiteNumérique";
+      after =
+        [ "network.target" ]
+        ++ (optional cfg.database.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-docs.service");
+      wants =
+        (optional cfg.database.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-docs.service");
+      wantedBy = [ "multi-user.target" ];
+
+      preStart = ''
+        ln -sf ${cfg.package}/lib/lasuite-docs/static /var/lib/lasuite-docs/
+
+        if [ ! -f .version ]; then
+          touch .version
+        fi
+
+        if [ "${cfg.package.version}" != "$(cat .version)" ]; then
+          ${getExe cfg.package} migrate && echo -n "${cfg.package.version}" > .version
+        fi
+      '';
+
+      script = ''
+        ${pythonPreloadSecrets}
+
+        ${cfg.package}/bin/gunicorn -c ${gunicornSettings} impress.wsgi:application
+      '';
+
+      environment = cfg.config;
+
+      serviceConfig = {
+        EnvironmentFile = optional (cfg.environmentFile != null) cfg.environmentFile;
+      } // commonServiceConfig;
+    };
+
+    systemd.services.lasuite-docs-celery = {
+      description = "Docs Celery broker from SuiteNumérique";
+      after =
+        [ "network.target" ]
+        ++ (optional cfg.database.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-docs.service");
+      wants =
+        (optional cfg.database.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-docs.service");
+      wantedBy = [ "multi-user.target" ];
+
+      script = ''
+        ${pythonPreloadSecrets}
+
+        ${cfg.package}/bin/celery -A impress.celery_app worker
+      '';
+
+      environment = cfg.config;
+
+      serviceConfig =
+        (optionalAttrs (cfg.environmentFile != null) { EnvironmentFile = cfg.environmentFile; })
+        // commonServiceConfig;
+    };
+
+    systemd.services.lasuite-docs-collaboration-server = {
+      description = "Docs Collaboration Server from SuiteNumérique";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      script = ''
+        ${optionalString (
+          cfg.collaborationServer.serverSecretPath != null
+        ) "export COLLABORATION_SERVER_SECRET=$(cat ${cfg.collaborationServer.serverSecretPath})"}
+        ${optionalString (
+          cfg.collaborationServer.yproviderApiKeyPath != null
+        ) "export Y_PROVIDER_API_KEY=$(cat ${cfg.collaborationServer.yproviderApiKeyPath})"}
+
+        ${cfg.collaborationServer.package}/bin/docs-collaboration-server
+      '';
+
+      environment = {
+        PORT = toString cfg.collaborationServer.port;
+        COLLABORATION_BACKEND_BASE_URL = "https://${cfg.domain}";
+      } // cfg.collaborationServer.config;
```

Comment:
These settings are not discoverable and should go as options into `services.lasuite-docs.collaborationServer.settings`.